stl/inc/experimental/filesystem

```diff
@@ -757,7 +757,7 @@ public:
         *this /= _Path_cvt<_Valty, value_type>::_Cvt(_Str_out, _Str.c_str(), _Str.size());
     }
 
-    template <class _InIt, enable_if_t<_Is_iterator_v<_InIt>, int> = 0>
+    template <class _InIt, enable_if_t<!is_same_v<_InIt, path>, int> = 0, enable_if_t<_Is_iterator_v<_InIt>, int> = 0>

```

Comment:
Do you mean `conjunction_v` at least that is guaranteed to short circuit

Reply:
This actually guarantees short-circuiting: " The substitution proceeds in lexical order and stops when a condition that causes deduction to fail is encountered." ([[temp.deduct]/7](http://eel.is/c++draft/temp.deduct#7.sentence-3)). The fix is ugly because I this header is deprecated and I was under the impression that we didn't have a deferred wrapper for `_Is_iterator_v`. I see now that we do have such a wrapper, so I'll `conjunction_v` here instead.